graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/CodeGen.kt

```diff
@@ -156,9 +160,29 @@ class CodeGen(private val config: CodeGenConfig) {
                 }
             }
         }
+
         return document
     }
 
+    private fun loadTypeMappingsFromDependencies() {
+        // process type mappings from dependencies
+        config.schemaJarFilesFromDependencies.forEach { file ->
+            val typeMappingFile = JarFile(file).getJarEntry("META-INF/dgs.codegen.typemappings")
+            if (typeMappingFile != null) {
+                val zipFile = ZipFile(file)
```

Comment:
Think we can just use the `jarFile` we already created above here instead of creating another `ZipFile` instance.